galaxy.yml

```diff
@@ -24,3 +24,11 @@ build_ignore:
 # https://docs.ansible.com/ansible/devel/dev_guide/developing_collections_distributing.html#ignoring-files-and-folders
   - .gitignore
   - changelogs/.plugin-cache.yaml
+  - codecov.yml
+  - Makefile
+  - Pipfile*
+  - "*.tar.gz"
+  - renovate.json
+  - .pylint.rc
+  - ansible.cfg
+  - .idea
```

Comment:
I would add ```.github``` as well.

Reply:
Applied in 129a2777d42645e183b696abcf70ad8b042b8b7d